libs/core/langchain_core/language_models/llms.py

```diff
@@ -101,7 +101,9 @@ def _before_sleep(retry_state: RetryCallState) -> None:
                 try:
                     loop = asyncio.get_event_loop()
                     if loop.is_running():
-                        loop.create_task(coro)
+                        # TODO: Fix RUF006 - this task should have a reference
+                        #  and be awaited somewhere
+                        loop.create_task(coro)  # noqa: RUF006
```

Comment:
There is something wrong here.
Do you have guidance in how to fix ? Or should I open an issue ?

Reply:
Could you open an issue and tag me and add `core` tag to it?

Reply:
I forgot to tell but I opened https://github.com/langchain-ai/langchain/issues/30791 for this.